apcd-cms/src/apps/utils/apcd_database.py

```diff
@@ -690,14 +689,14 @@ def create_other_exception(form, sub_data):
             sub_data[2],
             sub_data[3],
             _clean_value(form['requestor-name']),
-            _clean_email(form['requestor-email']),
+            form['requestor-email'],
```

Comment:
I think the more secure way to solve this problem would be to prevent users from submitting emails that will fail the clean check in the form, similar to how we prevent users from submitting unformatted phone numbers.